### DISCUSSION TITLE
Benchmark comparisons with latest Speedometer 3.0 between Chromium based browsers

### DISCUSSION THREAD

**Discussion by Girofox** [Alex313031/Thorium-Win#185] [General]
### Results
I used HWiNFO64 for monitoring IA core CPU power (which excludes GPU and DRAM power), there i got the average over a sample period of 500 with an 1 second polling rate.

Baseline IA power on idle is around 0.4 W.

No extension were running and the browser window was always focused.

### Thorium

```
AVX:	7.30 ± 0.16 	Geomean: 137.07	± (2.2%) 3.01 ms,  3.93 W

AVX2:	7.24 ± 0.27 	Geomean: 138.46	± (4.1%) 5.69 ms,  3.90 W

SSE3:	7.30 ± 0.27 	Geomean: 137.26	± (3.1%) 4.20 ms,  4.04 W
```
So it seems that i got a tiny less score with **AVX2** build (despite capable CPU) but IA core power usage was the lowest with **AVX2**.

In the Mercury repo i posted some benchmarks too where i got almost double the power usage for a similar score (why actually?) https://github.com/Alex313031/Mercury/discussions/158

### Comparison with stock chromium browsers
This is interesting and i first expected Brave at the top because it was founded by the creator of **JavaScript** itself (!) Brendan Eich.

At least on Android Brave still scores the most, even more than Thorium.

But since the latest several updates the V8 engine in Chrome really got improvements and consistently beats all other Chromium browsers on my Windows 11 setup (well, except Thorium which is the winner 😉)
### Brave Beta
- Score 6.85 @ 4.46 W (average)

`6.85 ± 0.34 	Geomean: 146.72	± (5.3%) 7.81 ms,  4.46 W`

### Chrome Beta

- Score 7.21 @ 4.17 W (average)

`7.21 ± 0.28 	Geomean: 139.15	± (4.4%) 6.16 ms,  4.17 W`

### MS Edge
- Score 6.87 @ 4.40 W (average)

`6.87 ± 0.21 	Geomean: 145.76	± (3.4%) 4.89 ms,  4.40 W`

**Comment by ZX-Commodore-ST** [comment]
My own results are less impressive. AMD Ryzen 2500U. Clean profiles, Thorium with ublock & google docs disabled. 100 iterations as I saw there is a big score difference when just doing a single test. https://browserbench.org/Speedometer3.0/?developerMode=&iterationCount=100 

- Chromium Hibbiki = 6.347
- Firefox Developer = 6.575
- Thorium SSE3 = 6.131
- Thorium AVX = 6.017
- Thorium AVX2 = 6.063
- Mercury SSE4 = 6.168
- Mercury AVX2 = 6.099

**Comment by Girofox** [comment]
Never heard from Chromium Hibbiki before, that looks impressive.
Hard to tell in my case if SSE4 or AVX2 is faster, my CPU is quiet old.

**Comment by ZX-Commodore-ST** [comment]
I will try again to test it on my desktop PC but right now I came into conclusion that my laptop is a very bad platform for benchmarking as it's unreliable. I don't have anything running in the background etc, I make sure it's always starting when CPU is cool. 100 iteration test takes about 30 minutes and sometimes I get random good results when other time I don't. I got 8.573 and 6.243 with RobRich compilation of Chromium. I got 6.347 and 5.858 with Hibbiki compilation of Chromium. I got 7.615 and 5.726 with Brave.

Performance is not really important factor as far as I'm concerned. I will keep following Thorium, Supermium and Brave as they are the only browsers that allow real portable use between users and PCs while still being able to use Chrome Web Store.
